src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -1124,6 +1124,24 @@ class CommonVisitor : public AST::BaseVisitor<Struct> {
     }
 
 
+    ASR::asr_t* make_dummy_assignment(ASR::expr_t* expr, const Location &loc) {
```

Comment:
`loc` isn't required probably as you can re-use `expr->loc`.

Reply:
I was slightly concerned just in case the `loc` stored a different value when used inside `make_call_helper()`. I will reuse the `expr->loc`. Thank you.